wd_helper.au3

```diff
@@ -920,7 +924,7 @@ Func _WD_jQuerify($sSession, $sjQueryFile = Default, $iTimeout = Default)
 
 	Local $iErr = @error
 
-	Return SetError(__WD_Error($sFuncName, $iErr))
+	Return SetError(__WD_Error($sFuncName, $iErr), 0)
```

Comment:
This change shouldn't be needed.

Reply:
[_WD_jQuerify Extended removed](https://github.com/Danp2/au3WebDriver/pull/290/commits/efe7fdbc70b329b692d1e54521cd2af109bd2285)